docs/development_suite/webhooks-and-events/events.mdx

```diff
@@ -91,79 +97,697 @@ The event is triggered as soon as the service's repository is successfully creat
+        "providerId": { "type": "string" },
+      },
+    },
+    "repositoryUrl": { "type": "string" },
+    "tenantId": { "type": "string" },
+  }
+}
+```
+
+</TabItem>
+
+</Tabs>
+
+### Service
+
+<Tabs>
+<TabItem value="service-table" label="Entity">
+
+|                Property                |   Type   |                                                                       Description                                                                       |
+| -------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `containerPorts`                       | `array`  | The container port                                                                                                                                      |
+| `defaultArgs`                          | `array`  | The default container args                                                                                                                              |
+| `defaultConfigMaps`                    | `array`  | The default config maps configured by marketplace item                                                                                                  |
+| `defaultDocumentationPath`             | `string` | The name of the service inside the project configuration                                                                                                |
+| `dockerImage`                          | `string` | The service url of docker image                                                                                                                         |
+| `defaultLogParser`                     | `string` | The [Log parser](./../../getting_started/monitoring-dashboard/dev_ops_guide/log.md#log-parsers)                                                         |
+| `defaultMonitoring`                    | `object` | The object which describes the [monitoring tools supported](./../../development_suite/api-console/api-design/microservice-monitoring.md) by the project |
+| `defaultEnvironmentVariables`          | `array`  | The environment variables configured by marketplace item                                                                                                |
+| `defaultProbes`                        | `object` | The default [probes](./../../development_suite/api-console/api-design/microservice-runtime-resources.md#probes) used by containers                      |
+| `defaultResources`                     | `object` | The default resources configured by marketplace item                                                                                                    |
+| `defaultSecrets`                       | `array`  | The default secrets configured by marketplace item                                                                                                      |
```

Comment:
Also for this

Reply:
i add inside the example the struct that the user can use